src/App.js

```diff
@@ -0,0 +1,11 @@
+import Calculator from './components/Calculator';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
```

Comment:
- [ ] The App.js component should be class-based, please change it in order to comply with the project requirements.